strider/config.py

```diff
@@ -10,7 +10,7 @@ class Settings(BaseSettings):
     kpregistry_url: AnyUrl = "https://kp-registry.renci.org"
     omnicorp_url: AnyUrl = "http://robokop.renci.org:3210"
     biolink_url: AnyUrl = "https://bl-lookup-sri.renci.org"
-    normalizer_url: AnyUrl = "https://nodenormalization-sri-dev.renci.org/1.1"
+    normalizer_url: AnyUrl = "https://nodenormalization-sri-prod.renci.org/1.1"
```

Comment:
Is this the right URL? I thought that this was the one: https://nodenormalization-sri.renci.org/docs